packages/netlify-cms-widget-object/src/ObjectControl.js

```diff
@@ -141,7 +144,15 @@ export default class ObjectControl extends React.Component {
                   onCollapseToggle={this.handleCollapseToggle}
                 />
               )}
-              {collapsed ? null : this.renderFields(multiFields, singleField)}
+              <div
+                className={cx({
+                  [css`
+                    ${styleStrings.collapsedObjectControl}
+                  `]: collapsed,
+                })}
+              >
+                {this.renderFields(multiFields, singleField)}
```

Comment:
Also a performance fix here, previously any  time the `object` widget toggles, it discards the old form and renders a new one.

Reply:
Nice fix 👍